frontend/src/components/templates/Temp1.js

```diff
@@ -23,39 +25,27 @@ const Temp1 = () => {
     OurBestMemory: "",
     confession: "",
   });
-  const imageURL = () => {
-    <CloudinaryContext
-      cloudName={process.env.cloudName}
-      apiKey={process.env.apiKey}
-      apiSecret={process.env.apiSecret}
-    >
-      <Image publicId={slamData.image} width="300" height="300" crop="fill" />
-    </CloudinaryContext>;
-    
-  };
   const handleSubmit = async (e) => {
     e.preventDefault();
-    console.log(slamData);
-    console.log(user);
-    imageURL();
-    // const response = await fetch(
-    //   "http://localhost:8000/digislam/apis/user",
-    //   {
-    //     method: "PUT",
-    //     headers: {
-    //       "Content-type": "application/json",
-    //       "Authorization": `Bearer ${user.token}`,
-    //     },
-    //     body: JSON.stringify(slamData),
-    //   }
-    // );
-    // const json = await response.json();
-    // if (response.ok) {
-    //   console.log(json);
-    // }
-    // if (!response.ok) {
-    //   console.log(json.error);
-    // }
+    const response = await fetch(
+      `http://localhost:8000/digislam/apis/users/${user.user._id}`,
```

Comment:
Add `http://localhost:8000` in `.env` later when you have to deploy it will become easier to change it later for all the files where you have used it.